roles/rabbitmq/handlers/main.yml

```diff
@@ -1,2 +1,8 @@
-  - name: restart rabbitmq
-    action: service name=rabbitmq-server state=restarted enabled=yes
+---
+- name: restart rabbitmq
+  action: service name=rabbitmq-server state=restarted enabled=yes
```

Comment:
While you're changing the file, might as well fix this to remove the `action` bit.